### DISCUSSION TITLE
column header in the bold text identification

### DISCUSSION THREAD

**Discussion by ashokrayapati** [jsvine/pdfplumber#1174] [Ask for help with specific PDFs]
Hello Team,

First of all thank you for the wonderful library provided to the users to handle multiple pdfs

i have multiple tables from page.find_tables() and if tables are there then i need to identify the text for the row as column header or column names if the text is bold.

can you please help on this in the logic

**Comment by jsvine** [comment]
Hi @ashokrayapati — you can try something like this approach: https://github.com/jsvine/pdfplumber/discussions/1106#discussioncomment-8765159

More generically: 

- You'd loop through `Table.cells`
- Crop the page to each cell's bounding box
- Use the approach in the comment above *or* use something like `cropped.extract_words(extra_attrs="fontname")`
